.jenkins/pytorch/test.sh

```diff
@@ -42,14 +42,8 @@ if [[ "$BUILD_ENVIRONMENT" != *ppc64le* ]]; then
   # TODO: move this to Docker
   pip install -q hypothesis --user
 
+  pip install -q tb-nightly
```

Comment:
Let's not switch over to tb-nightly. We'll wait until 1.14 is available with `pip install tensorboard` to change this.